Basic-Car-Maintenance/Shared/Dashboard/Views/EditEventDetailView.swift

```diff
@@ -32,7 +32,7 @@ struct EditMaintenanceEventView: View {
                         }
                     } label: {
                         Text("Select a vehicle",
-                             comment: "Maintenance event vehicle picker label")
+                             comment: "Label for Picker for selecting a vehicle")
```

Comment:
```suggestion
                             comment: "Picker for selecting a vehicle")
```

All comments for a single key should be the same, this is something I've recently figured out about how Localizable.xcstrings works